templates/telemetry/pioneer-study/pioneer-study.1.parquetmr.txt

```diff
@@ -0,0 +1,20 @@
+message pioneer-study {
```

Comment:
The parquet output schemas should be study specific (the generic version isn't necessary at all)
i.e. templates/pioneer-study/foobar.1.parquetmr.txt